src/components/UniversalNavbarExpanded/UniversalNavbarExpanded.js

```diff
@@ -166,6 +180,8 @@ UniversalNavbarExpanded.propTypes = {
       })
     ),
   }).isRequired,
+  /** Estimate copy experiment on optimizely */
+  estimateExperiment: PropTypes.bool,
```

Comment:
Would it be possible to have this component take an optional `children` or `experimentChildren` and leave the component content, design, and optimizely tracking up to the consumer of `UniversalNavbarExpanded`?

Reasoning behind this is that it seems like we're slowly leaning towards allowing consumers of EDS to specify className and other properties that were previously hardcoded. The UniversalNavbar is still pretty opinionated, but it could be nice to slowly refactor it to support more behaviors.

Reply:
@stephaniel96 

If I had more time I would be willing to make the switch to passing in children. We're trying to get this experiment turned on tomorrow and I've still got some loose ends CMS side to tidy. 

This `Expanded` version is already specific to CMS only and this `estimateExperiment` prop is still optional, should another consumer need to use the navbar and not worry about specifying a value here.

I'd be willing to make a growth backlog ticket here, to address your request when I have more bandwidth, so that future consumers can pass in children or classes. 

This experiment code should only be temporary and removed once we see a winner in Optimizely.

Reply:
Thanks for your explanation! I didn't realize that this was a specific `Expanded` version, so that's an extra compelling reason 👍